spec-0000/index.md

```diff
@@ -33,12 +34,26 @@ Specifically, we recommend that:
 1. Support for Python versions be dropped **3 years** after their initial release.
 2. Support for core package dependencies be dropped **2 years** after their initial release.
 
+{{< admonition note >}}
+Core packages may or may not to provide bug fix releases during the full 2 year period after release.
+So projects may decide to occassionally drop support for core pakage dependencies earlier than recommended by this SPEC.
+For instance, if a newer minimum version of a core package is needed by a project due to a critical bug fix,
+which is not backported to older versions.
+{{< /admonition >}}
+
 ### Core Project Endorsement
 
 <!--
 Briefly discuss what it means for a core project to endorse this SPEC.
 -->
 
+Core project endorsing this SPEC means that those projects encourage all projects across the Scientific Python ecosystem
+to limit how long they support older Python versions and older core packages versions.
+While core projects may wish to provide bug-fix releases of older versions, endorsing this SPEC does **not** imply that
+core projects will provide bug-fix releases during the full 2 year period after release.
+As many of the core projects are volunteer projects with limited developer time, core package developers may often prioritize
+new features and enhancements over long-term package maintenance.
```

Comment:
I don't think we should add that note. For some projects that could be argued and here that would not really help the messaging IMHO. At most I would say something along the line of "project's developments are discussed and prioritised with and by the community".

Reply:
I agree that it's better without the last sentence.